cypress/integration/plugins/dashboards-flow-framework/create_workflow_spec.js

```diff
@@ -0,0 +1,209 @@
+    cy.getElementByDataTestId('importWorkflowButton')
+      .should('be.visible')
+      .click();
+    cy.contains('Import a workflow (JSON/YAML)').should('be.visible');
+    const filePath =
+      'cypress/fixtures/' + FF_FIXTURE_BASE_PATH + 'semantic_search_query.json';
+    cy.get('input[type=file]').selectFile(filePath);
+    cy.contains('The uploaded file is not a valid workflow').should(
+      'be.visible'
+    );
+  });
+
+  it('create workflow using Semantic Search template', () => {
+    cy.contains('h2', 'Semantic Search', { timeout: 120000 })
+      .should('be.visible')
+      .parents('.euiCard')
+      .within(() => {
+        cy.contains('button', 'Go').click();
+      });
+    cy.getElementByDataTestId('optionalConfigurationButton')
+      .should('be.visible')
+      .click();
+    cy.getElementByDataTestId('selectDeployedModel')
+      .should('be.visible')
+      .click();
+    cy.get('.euiSuperSelect__item').should('be.visible');
+    cy.get('.euiSuperSelect__item').contains('BedRock').click();
+    cy.contains('label', 'Text field')
+      .invoke('attr', 'for')
+      .then((id) => {
+        cy.get(`#${id}`).clear().type('item_text');
+      });
+    cy.getElementByDataTestId('quickConfigureCreateButton')
+      .should('be.visible')
+      .click();
+    cy.url().should('include', '/workflows/');
+    cy.getElementByDataTestId('editSourceDataButton')
+      .should('be.visible')
+      .click();
+    cy.getElementByDataTestId('uploadSourceDataButton')
+      .should('be.visible')
+      .click();
+    const filePath = `cypress/fixtures/${FF_FIXTURE_BASE_PATH}/semantic_search_source_data.json`;
+    cy.get('input[type=file]').selectFile(filePath);
+    cy.getElementByDataTestId('closeSourceDataButton')
+      .should('be.visible')
+      .click();
+    cy.mockIngestion(() => {
+      cy.getElementByDataTestId('runIngestionButton')
+        .should('be.visible')
+        .click();
+    });
+    cy.getElementByDataTestId('searchPipelineButton')
+      .should('be.visible')
+      .click();
+    cy.getElementByDataTestId('queryEditButton').should('be.visible').click();
+    cy.get('[data-testid="editQueryModalBody"]').within(() => {
+      cy.fixture(FF_FIXTURE_BASE_PATH + 'semantic_search_query.json').then(
+        (jsonData) => {
+          const jsonString = JSON.stringify(jsonData);
+          cy.get('.ace_text-input')
+            .focus()
+            .clear({ force: true })
+            .focus()
+            .wait(2000)
+            .type(jsonString, {
+              force: true,
+              parseSpecialCharSequences: false,
+              delay: 5,
+            })
+            .trigger('blur', { force: true });
+        }
+      );
+    });
+    cy.getElementByDataTestId('searchQueryCloseButton')
+      .should('be.visible')
+      .click();
+    cy.mockSearchIndex(() => {
+      cy.getElementByDataTestId('runQueryButton').should('be.visible').click();
+    });
```

Comment:
As a final check, can we check that the mocked search response is visible in the Tools panel? Same goes for ingest.